React/Base/RCTRootView.m

```diff
@@ -366,6 +391,35 @@ - (void)contentViewInvalidated
   [self showLoadingView];
 }
 
+- (void)maybeUpdateLayoutContext
+{
+  CGRect frame = self.frame;
+  UIEdgeInsets safeAreaInsets = RCTSafeAreaInsetsForView(self, YES);
+  if (
+    RCTUIEdgeInsetsEqualToEdgeInsetsWithThreshold(safeAreaInsets, _lastSafeAreaInsets, RCTScreenScale()) &&
+    CGRectEqualToRect(frame, _lastFrame)
```

Comment:
why not use threashold for both?

Reply:
The existing safe area view code used this when comparing safe area values so I kept it. For frames it does not seem to be needed, I did a quick search for `threshold` and we don't seem to do anything special when comparing frames. Not sure why we need `RCTUIEdgeInsetsEqualToEdgeInsetsWithThreshold` in the first place.